components/breadcrumbs/src/atoms/link/test.tsx

```diff
@@ -4,7 +4,10 @@ import { render } from '@testing-library/react';
 import Link from '.';
 
 describe('Breadcrumbs Link', () => {
-  it('renders without crashing', () => {
-    render(<Link href="#id">Example</Link>);
+  it('renders contents without crashing', () => {
+    const contents = 'Example';
+    const { getByText } = render(<Link href="#id">{contents}</Link>);
+
+    expect(getByText(contents)).toBeInTheDocument();
```

Comment:
minor - I would suggest for this test, from the user's perspective, they would expect to be able to click on the contents (even if nested dom contents) and the url would update to include #id

Reply:
in the process of writing this test from the users perspective, it would also test that it 'renders contents without crashing'

Reply:
@stevesims any thoughts on this and https://github.com/govuk-react/govuk-react/pull/909#discussion_r680850550 ?